protocol/messenger_handler.go

```diff
@@ -2880,7 +2880,7 @@ func mapSyncAccountToAccount(message *protobuf.SyncAccount, accountOperability a
 		PublicKey: types.HexBytes(message.PublicKey),
 		Path:      message.Path,
 		Name:      message.Name,
-		Color:     message.Color,
```

Comment:
@Khushboo-dev-cpp this change here will break wallet accounts' color for all Status users who are recovering from waku (color for backed up account is under field `Color`, but we're trying to get it from field `ColorID`, also even if the name is set we need a mapper to map old colors to a new values). 

In general I am not sure do we need to be a backward compatible at this stage (before mvp) if we need, then we need that handler as well.

Reply:
Shall we just have a default color? if not provided?

Reply:
> Shall we just have a default color? if not provided?

If we need to be a backward compatible at this point we would need to map "old" `color` value to a "new" `colorID`, but since we are still in the development phase I wouldn't add the code for handling that and make it more robust, at least not before mvp, especially having in mind that a very few people will recover from waku before mvp.